google_guest_agent/instance_setup.go

```diff
@@ -265,7 +266,10 @@ func generateSSHKeys(ctx context.Context) error {
 			logger.Warningf("Generated key is malformed, not uploading")
 		}
 	}
-	run.Quiet(ctx, "restorecon", "-FR", hostKeyDir)
+	_, err = exec.LookPath("restorecon")
+	if err == nil {
+		run.Quiet(ctx, "restorecon", "-FR", hostKeyDir)
```

Comment:
i think we should at-least log error if any

Reply:
Nope, it's actually an error condition. I'll make it return an error.

Reply:
done. thanks.